### PR TITLE
servicebusexplorer: Update to version 5.0.5

### PR DESCRIPTION
**Pull request by 0xced**

### CODE REVIEW

File: bucket/servicebusexplorer.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+    "version": "5.0.5",
     "description": "Connect to a Service Bus namespace and efficiently administer messaging entities.",
     "homepage": "https://github.com/paolosalvatori/ServiceBusExplorer",
     "license": "Apache-2.0",
-    "url": "https://github.com/paolosalvatori/ServiceBusExplorer/releases/download/5.0.4/servicebusexplorer.5.0.4.nupkg",
-    "hash": "9d4db11dfa2d6e880d2dc3e974c0a11081ba4a54820368120fe6e24a4135d724",
-    "bin": "tools\\servicebusexplorer.exe",
+    "url": "https://github.com/paolosalvatori/ServiceBusExplorer/releases/download/5.0.5/servicebusexplorer.5.0.5.nupkg",
+    "hash": "e2fe0369e7acb6ec7c92a2785f3dc62c471720715551b3263d5392b7b6fbaf16",
+    "bin": "tools\\net462\\servicebusexplorer.exe",
```
Comment (rashil2000) [inline bucket/servicebusexplorer.json:8]:
```suggestion
    "extract_dir": "tools\\net462",
```

File: bucket/servicebusexplorer.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
     "shortcuts": [
         [
-            "tools\\servicebusexplorer.exe",
+            "tools\\net462\\servicebusexplorer.exe",
```
Comment (rashil2000) [inline bucket/servicebusexplorer.json:11]:
```suggestion
            "servicebusexplorer.exe",
```

**Comment by rashil2000** [comment]
Please resolve conflicts

**Comment by 0xced** [comment]
The bot already took care of updating the manifest to the latest released version in 3400fc0ebeb370ad031b73f7aa5983721a140e0a rendering this pull request moot.

**Pull request closed without merging**